app/scripts/SeriesListItems.js

```diff
@@ -91,7 +91,7 @@ export const getSeriesItems = (
         <span styleName="context-menu-span">
           {x.options && x.options.name && x.options.name.length
             ? x.options.name
-            : x.uid}
+            : x.type}
```

Comment:
Just curious, does this fix a bug? Seems unrelated to the coloring.

Reply:
Can't get anything past you ;-) Yeah, it fixes a small bug.